lib/helper.js

```diff
@@ -176,9 +176,10 @@ class Helper {
    * @param {!NodeJS.EventEmitter} emitter
    * @param {(string|symbol)} eventName
    * @param {function} predicate
+   * @param {!Promise<!Error>} abortPromise
```

Comment:
not you, but this method is missing a JSDoc for timeout. We should really lint for that.

Reply:
Done. Added a type annotation for this.